runtime/onert/core/src/exec/LinearExecutor.cc

```diff
@@ -49,9 +49,26 @@ void LinearExecutor::executeImpl()
       fn_seq->enableDynamicShapeInferer(handle_dynamic_tensor);
       fn_seq->run();
 
-      _subject.notifyJobEnd(this, profiling_subg_index, code.op_ind, backend);
+      _subject.notifyJobEnd(this, subg_idx, code.op_ind, backend);
+
+      auto op_idx = code.op_ind;
+      auto &backend_ctx = _backend_contexts.at(backend);
+      const auto &tensor_reg = backend_ctx->tensor_registry;
+      const auto &op = _graph.operations().at(op_idx);
+      const auto &outputs = op.getOutputs();
+      if (outputs.size() != 1)
+        throw std::runtime_error("Only 1 output operator is supported for recording minmax.");
+      auto out_tensor = tensor_reg->getITensor(outputs.at(0));
+      auto opcode = code.op->opcode();
+      notifyExecuteOpEnd(subg_idx, op_idx, opcode, out_tensor);
```

Comment:
NOTE: `notifyExecuteOpEnd` signature should be changed for multiple outputs. https://github.com/Samsung/ONE/pull/10607#issuecomment-1499893984